### PR TITLE
update the readme file for remainingTime feature

### PR DESCRIPTION
**Pull request by Amir-Alipour**
Hi,
I updated the doc for the new feature and fixed the Legal id short link problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -415,6 +416,27 @@ timeAgo('1400/04/07 18:00:00') // حدود 3 هفته  بعد
...
+
+**Usage**
+
+>Takes a date(it could be string, number or date) and calculate years,
+>months, days, hours, minutes and seconds remained to that specific date. 
+
+```js 
+import { remainingTime } from "@persian-tools/persian-tools";
+
+// CurrentDateTime: () => new Date("2022-04-12T10:30:51Z")
```
Comment (ali-master) [inline README.md]: Remove the unused line

File: README.md [hunk 2]
```diff
@@ -415,6 +416,27 @@ timeAgo('1400/04/07 18:00:00') // حدود 3 هفته  بعد
...
+
+remainingTime("2023-05-14T13:35:59Z").toString() // ۱ سال و ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
+remainingTime("2022-05-14T13:35:59Z").toString() // ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
+remainingTime("2022-04-14T13:35:59Z").toString() // ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه
```
Comment (ali-master) [inline README.md]: Please write other methods and results such as `RemainingTime` object and `IsFinished`

**Review by ali-master — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ali-master) [reply README.md]: And there is no need to write more than 4 samples.

File: README.md (hunk 1 above)
Comment (Amir-Alipour) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (Amir-Alipour) [reply README.md]: check the changes

File: README.md [hunk 3]
```diff
@@ -415,6 +416,26 @@ timeAgo('1400/04/07 18:00:00') // حدود 3 هفته  بعد
...
+>Takes a date(it could be string, number or date) and calculate years,
+>months, days, hours, minutes and seconds remained to that specific date. 
+
+```js 
+import { remainingTime } from "@persian-tools/persian-tools";
+
+remainingTime("2023-05-14T13:35:59Z").toString() // ۱ سال و ۱ ماه و ۲ روز و ۳ ساعت و ۵ دقیقه و ۸ ثانیه 
+
+const { years, months, days, hours, minutes, seconds, isFinished } = remainingTime("2023-05-14T13:35:59Z");
+console.log(years) // 1
```
Comment (ali-master) [inline README.md]: Remove `console.log`s

**Review by ali-master — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Amir-Alipour) [reply README.md]: removed

**Review by ali-master — APPROVED** [review]

**Comment by ali-master** [comment]
@all-contributors please add @Amir-Alipour for doc

**Comment by ali-master** [comment]
@all-contributors please add @Amir-Alipour for doc

**Pull request merged**